cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -740,15 +740,12 @@ def launch_default_ingress_controller():
     # Render the ingress daemon set controller manifest
     context['ingress_image'] = config.get('nginx-image')
     if context['ingress_image'] == "" or context['ingress_image'] == "auto":
-        if context['arch'] == 's390x':
-            context['ingress_image'] = \
-                "docker.io/cdkbot/nginx-ingress-controller-s390x:0.9.0-beta.13"
-        elif context['arch'] == 'arm64':
-            context['ingress_image'] = \
-                "k8s.gcr.io/nginx-ingress-controller-arm64:0.9.0-beta.15"
-        else:
-            context['ingress_image'] = \
-                "k8s.gcr.io/nginx-ingress-controller:0.9.0-beta.15"  # noqa
+        images = {'amd64': 'quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.15.0', # noqa
+                  'arm64': 'quay.io/kubernetes-ingress-controller/nginx-ingress-controller-arm64:0.15.0', # noqa
+                  's390x': 'quay.io/kubernetes-ingress-controller/nginx-ingress-controller-s390x:0.15.0', # noqa
+                  'ppcle': 'quay.io/kubernetes-ingress-controller/nginx-ingress-controller-ppc64le:0.15.0', # noqa
```

Comment:
`ppcle` looks suspect. I'm pretty sure the dictionary key needs to be `ppc64el` because the key comes from `dpkg --print-architecture` ([link](https://github.com/hyperbolic2346/kubernetes-1/blob/a3ec55e3dfb5605761bf9a3f9802cc9ef41f752d/cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py#L529)), which is the arch as defined by debian, which is `ppc64el` ([link](https://www.debian.org/ports/)).

The `ppc64le` in the image URL looks correct though :+1: